tests/test_plots.py

```diff
@@ -54,14 +54,14 @@ def test_compare_flights(mock_show, calisto, example_env):
         Mocks the matplotlib.pyplot.show() function to avoid showing the plots.
     calisto : rocketpy.Rocket
         Rocket object to be used in the tests. See conftest.py for more details.
-    example_env : rocketpy.Environment
+    example_plain_env : rocketpy.Environment
         Environment object to be used in the tests. See conftest.py for more details.
 
     Returns
     -------
     None
     """
-    example_env.set_atmospheric_model(
+    example_plain_env.set_atmospheric_model(
         type="custom_atmosphere",
         pressure=None,
         temperature=300,
```

Comment:
I believe in the future we should be using a fixture with the custom atmosphere already set. This would avoid us to set the atm model several times, which is really time consuming.

Reply:
I agree this would increase readability by avoiding code repetition. However I could not find a common base for the atmospheric model used in tests, sometimes custom_atm is used, sometimes standard, sometimes other, etc...

Also, I'm not sure that moving it to fixtures will speed up testing as each fixture call creates a new instance to set the atm model (fixture instances are not shared between tests).

Reply:
I searched in the code and found 2 instances of "example_plain_env.set_atmospheric_model("custom_atmosphere"...." (not written exactly like this) in the tests.

*Probably* a fixture that takes the plain environment and set the custom_atmosphere would be useful.
No need to implement it in this PR tho.